src/codemagic/cli/argument/argument_properties.py

```diff
@@ -12,13 +12,19 @@
 from typing import Union
 
 
+class MutuallyExclusiveGroup(NamedTuple):
+    group_name: str
```

Comment:
Just for the sake of simplicity, I think we can use just `name` instead of `group_name`.
```suggestion
    name: str
```